clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -2053,26 +2053,26 @@ inline ::std::pair<Iter1, Iter1> equal_range(_ExecutionPolicy &&policy,
 // DplExtrasVector|is_iterator
 // DPCT_DEPENDENCY_END
 // DPCT_CODE
-template <typename _ExecutionPolicy, typename key_t, typename key_out_t, 
+template <typename _ExecutionPolicy, typename key_t, typename key_out_t,
           typename OffsetIteratorT>
 inline ::std::enable_if_t<dpct::internal::is_iterator<key_t>::value &&
                           dpct::internal::is_iterator<key_out_t>::value>
 segmented_reduce_argmin(_ExecutionPolicy &&policy, key_t keys_in,
-                        key_out_t keys_out, ::std::int64_t n,
-                        ::std::int64_t nsegments, OffsetIteratorT begin_offsets,
+                        key_out_t keys_out, ::std::int64_t nsegments,
+                        OffsetIteratorT begin_offsets,
                         OffsetIteratorT end_offsets) {
   policy.queue().submit([&](sycl::handler &cgh) {
     cgh.parallel_for(nsegments, [=](sycl::id<1> i) {
       ::std::int64_t segment_begin = begin_offsets[i];
```

Comment:
I think we can use `std::iterator_traits<OffsetIteratorT>::value_type` instead of `std::int64_t`
```
using OffsetT = std::iterator_traits<OffsetIteratorT>::value_type;
OffsetT segment_begin = begin_offsets[i];
OffsetT segment_end = end_offsets[i];

if (segmented_end <= segmented_begin) {
...
}
```